src/init.cpp

```diff
@@ -1462,13 +1461,6 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
                     break;
                 }
 
-                // open sidechain db, read data and populate memory objects
```

Comment:
The sc db size 'nSideChainDBCache' is not used anymore (see above). Moreover the nCoinDBCache should be probably increased accordingly

Reply:
I agree, the whole sizing policy for the dbs may be affected by the decision of storing the sidechains into the chainstatedb. I'd move the investigation for the optimal sizing once we consolidate the sidechain informations we are going to persist and make sure the correctness checks out. Thanks